modules/react-components/src/components/footer/footer.tsx

```diff
@@ -127,7 +127,7 @@ export const Footer: FunctionComponent<FooterPropsInterface> = (
             borderless 
             data-testid={ testId }
         >
-            <Container  data-testid={ `${ testId }-container` } className="app-footer-container">
+            <Container fluid data-testid={ `${ testId }-container` } className="app-footer-container">
```

Comment:
This might make the `My Account` footer fluid as well. Would be better to only go with `fluid` is supplied as a prop.
```suggestion
            <Container fluid={ fluid } data-testid={ `${ testId }-container` } className="app-footer-container">
```